### PR TITLE
Prometheus metrics

### PR DESCRIPTION
**Pull request by thefearlesscoder**
Here is the client code for exposing metrics to prometheus on the `/metrics` endpoint

### CODE REVIEW

File: splash/__init__.py [hunk 1]
```diff
@@ -79,20 +84,28 @@ def create_app(db=None):
     from splash.auth.oauth_resources import OAuthResource
     api.add_resource(OAuthResource, "/api/tokensignin", resource_class_kwargs={'user_service': app.user_service})
 
+    @app.route('/metrics/')
+    def metrics():
+        return Response(prometheus_client.generate_latest(),
+                        mimetype=CONTENT_TYPE_LATEST)
+
```
Comment (thefearlesscoder) [inline splash/__init__.py:91]: Yes this is where the endpoint is added.

Comment (dylanmcreynolds) [reply splash/__init__.py:91]: Missed that. Great.

**Comment by dylanmcreynolds** [comment]
Before merging, I'd like to know what happens if this was installed without a prometheus server polling for metrics. Will something get really big? Does it flush by default or do we need to flush somehow? If that's not easy, then we need to add some sort of configuration option to turn this off in a running server.

**Comment by thefearlesscoder** [comment]
https://www.robustperception.io/memory-usage-of-prometheus-client-libraries This link addresses this problem.

**Pull request merged**